### DISCUSSION TITLE
Invalid API Key provided: sk_live_ in test gateway

### DISCUSSION THREAD

**Discussion by dirums** [swellstores/community#121] [Help] [answered]
Hi,
I'm checking the swell checkout process and during the payment stage I'm using the test gateway and using the test card for payment but I'm getting the "Invalid API Key provided: sk_live_" error.

Any solutions?

**Comment by greph** [comment]
Hey @dirums , can you check that the correct Stripe API key is being used (from the associated test environment). This error is commonly returned from Stripe when there is a mismatch.

**Reply by mustafahoda** [reply, marked as answer]
Hi @dirums ,

I had a similar issue last week. In addition to @greph response, I'd also disconnect the Payment Gateway and give that a shot to ensure you purge any incorrect settings that still may exist.
<img width="436" alt="image" src="https://user-images.githubusercontent.com/3252808/219070063-23a926c2-2961-4303-abbd-fb28a42f0af1.png">
